packages/mc-scripts/src/config/create-webpack-config-for-production.ts

```diff
@@ -495,6 +495,14 @@ function createWebpackConfigForProduction(
           exclude: /node_modules/,
           use: [require.resolve('graphql-tag/loader')],
         },
+        process.env.ENABLE_MESSAGE_LOADER === 'true' && {
+          test: /i18n\/data\/.*\.json$/,
```

Comment:
Open question: Should this be configurable somehow? Is it possible that consumers of the app-kit have their translations located somewhere else?

Reply:
Our [recommendation](https://docs.commercetools.com/merchant-center-customizations/development/translations#extracting-messages-for-translations) is to use the same path for Custom Applications authors, however they can do what they want.

I would say that we can keep the path static. The flow would work anyway and won't affect Custom Application authors which do not use that path.
Since we don't know how many there are, I would vote for keep it simple and revisit in case we learn about actual users require it.

@emmenko what do you think?

Reply:
I agree, let's go with this for now.